test-lib/src/main/AndroidManifest.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest
+<manifest xmlns:tools="http://schemas.android.com/tools"
     xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <permission android:name="android.permission.WAKE_LOCK"/>
+    <permission android:name="android.permission.WAKE_LOCK"
+        tools:ignore="ReservedSystemPermission" />
```

Comment:
What's this coming from?  As far as I know, this permission has yet to be restricted anywhere in Android.

Reply:
Yes, it's strange. The error was this: 
```
Lint found errors in the project; aborting build.
Error: android.permission.WAKE_LOCK is a reserved permission
```
I'll take a deeper look at it tomorrow.

Reply:
To be honest I can't find anything useful about this new behaviour, so I placed a question to the ASG.

Reply:
It seems to be related to my local environment, as it's alright here in CI tests.